drivers/drivers_full.py

```diff
@@ -446,3 +445,36 @@ def sleep(self):
         self.wait_until_idle()
         self.send_command(self.DEEP_SLEEP)
         self.send_data(0xA5)
+
+    def reset(self):
+        """
+        Mirroring behaviour in reference implementation:
+        https://github.com/waveshare/e-Paper/blob/702def06bcb75983c98b0f9d25d43c552c248eb0/RaspberryPi%26JetsonNano/python/lib/waveshare_epd/epd7in5_V2.py#L48-L54
+
+        The `reset` inherited from `WaveshareFull` did not work (`init` hanged at `wait_until_idle` after the `POWER_ON`
+        command was sent.
```

Comment:
Maybe for future reference it would be clearer to change this to something like:

```suggestion
        The earlier implementation of `reset` inherited from `WaveshareFull` did not work with some units 
        (`init` hanged at `wait_until_idle` after the `POWER_ON` command was sent).
```